modules/index.js

```diff
@@ -1,14 +1,30 @@
 const payloadPassThrough = (state, payload) => payload;
 
+const handlePayloadPassThrough = (actionSpec) => {
+  const maybeActionReducer = actionSpec.slice(-1)[0];
+  const allButLast = actionSpec.slice(0, -1);
+
+  if (typeof maybeActionReducer === 'function') {
+    return { actionTypes: allButLast, actionReducer: maybeActionReducer };
+  }
+  if (typeof maybeActionReducer === 'string') {
+    return { actionTypes: actionSpec, actionReducer: payloadPassThrough };
+  }
+  throw new Error('Reducer must either be a function or not present (payloadPassThrough)');
```

Comment:
I would rephrase to: `'[redux-action-reducer] Arguments of `createReducer` must either contain a reducer function or none at all (payload pass-through).'`